### PR TITLE
@/codeforafrica/components/ShareBar

### LINKED ISSUE #79: @/codeforafrica/components/ShareBar
Implement ShareBar component per designs.

Image

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Resolves #79 

**NOTE**: This PR fixes `@/codeforafrica` tests that were failing.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Screenshot 2022-05-11 at 16-14-48 Battle for gender equality in African media continues Stories Code for Africa

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by nerdlet — APPROVED** [review]
@kilemensi Looks good 🚀 🚀

File: apps/codeforafrica/src/pages/stories/[slug].page.js [hunk 1]
```diff
@@ -1,25 +1,68 @@
...
             sx={{
-              bgcolor: { xs: "inherit", sm: "background.main" },
+              color: "#B4ABAB",
+              columnGap: 2,
+              display: "flex",
+              alignItems: "center",
             }}
           >
-            <RelatedStories {...section} sx={{ px: { xs: "20px", sm: 0 } }} />
+            <Typography variant="footerCap">Share This Article</Typography>
```
Comment (gertie-sheshe) [inline apps/codeforafrica/src/pages/stories/[slug].page.js:37]: Currently displaying as `14px` It is `10px` on design

File: apps/codeforafrica/src/pages/stories/[slug].page.js [hunk 2]
```diff
@@ -1,25 +1,68 @@
...
+              sm: "648px",
+              md: "912px",
+            },
+          }}
+        >
           <Box
-            key={section.slug}
             sx={{
-              bgcolor: { xs: "inherit", sm: "background.main" },
+              color: "#B4ABAB",
```
Comment (gertie-sheshe) [inline apps/codeforafrica/src/pages/stories/[slug].page.js:31]: theme?

File: apps/codeforafrica/src/pages/stories/[slug].page.js (hunk 1 above)
Comment (kilemensi) [reply apps/codeforafrica/src/pages/stories/[slug].page.js:37]: .. And why might that be? It's clearly defined as `10px` here: https://github.com/CodeForAfrica/ui/pull/80/files#diff-7ef618251bfea0bfb75d63e8e168880ab55cdf287f444b09a10854575a22bc32R135

File: apps/codeforafrica/src/pages/stories/[slug].page.js (hunk 2 above)
Comment (kilemensi) [inline apps/codeforafrica/src/pages/stories/[slug].page.js:31]:
```suggestion
              color: "grey.main",
```

File: apps/codeforafrica/src/pages/stories/[slug].page.js (hunk 1 above)
Comment (kilemensi) [reply apps/codeforafrica/src/pages/stories/[slug].page.js:37]: Override was missing.

**Review by gertie-sheshe — APPROVED** [review]

**Pull request merged**